### PR TITLE
feat: gho bridge tx history

### PR DESCRIPTION
**Pull request by grothem**

### CODE REVIEW

File: src/components/transactions/Bridge/BridgeActions.tsx [hunk 1]
```diff
@@ -185,44 +186,19 @@ export const BridgeActions = React.memo(
           }
         );
 
-        const onRampInterface = new utils.Interface(onRampAbi);
+        // const onRampInterface = new utils.Interface(onRampAbi);
 
         const receipt = await sendTx.wait(1);
+        console.log('receipt', receipt);
```
Comment (foodaka) [inline src/components/transactions/Bridge/BridgeActions.tsx:192]: should we remove this and these comments if we are not using

File: src/components/transactions/Bridge/BridgeConfig.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+};
+
+export enum MessageExecutionState {
+  UNTOUCHED = 0,
+  IN_PROGRESS,
+  SUCCESS,
+  FAILURE,
+}
+
+// TODO: make testnet and mainnet config
```
Comment (foodaka) [inline src/components/transactions/Bridge/BridgeConfig.ts:22]: yeah can't forget also for supported networks as well

File: src/hooks/useBridgeTransactionStatus.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+      return MessageExecutionState.UNTOUCHED;
+    },
+    queryKey: ['executionState', chainId, sequenceNumber],
+    enabled: offRamps?.length > 0,
+    refetchInterval: (data) => {
+      if (data === MessageExecutionState.SUCCESS) {
+        return false;
+      }
+
```
Comment (foodaka) [inline src/hooks/useBridgeTransactionStatus.tsx:48]:
```suggestion
// Note: 60 seconds
```

File: src/components/transactions/Bridge/BridgeConfig.ts (hunk 2 above)
Comment (grothem) [reply src/components/transactions/Bridge/BridgeConfig.ts:22]: Yeah, I added a function in this file, `getSupportedSourceChains`, it just returns all of the chain ids that are in the config. So essentially if it's not in the lane config, it's not a supported network.

**Review by MareskoY — APPROVED** [review]

**Pull request merged**